webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/VariableAndSectionSpec.scala

```diff
@@ -136,10 +139,6 @@ final case class SectionSpec(
   private def recCloneMultivalued: Either[LoadTechniqueError, SectionSpec] = {
     val multivaluedChildren = children.toList.traverse { child => child match {
       case s: SectionSpec =>
-        if (s.isMultivalued) LoadTechniqueError.Consistancy(
```

Comment:
I think this isnot relevant anymore since it could prevent usage of group reporting

Reply:
I think you are right, cc @ncharles